apps/mobile-wallet/src/persistent-storage/wallet.ts

```diff
@@ -107,8 +240,22 @@ export const getStoredWallet = async (error?: string): Promise<WalletMetadata> =
   return walletMetadata
 }
 
+export const getStoredWalletMetadataWithoutThrowingError = async (): Promise<WalletMetadata | null> => {
+  let walletMetadata = null
+
+  try {
+    walletMetadata = await getWalletMetadata()
```

Comment:
Mhh it will actually throw no? 
In the `getWalletMetadata` definition you set  `throwError = true`. So is the parameter is omited, like above, errors will be thrown.

Reply:
Also, I should probably continue to review to see where this is used, but why not directly use `getWalletMetadata(false)`?

Reply:
Fixed